lib/screens/second_screen.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:dio/dio.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_localization/data/album.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class SecondScreen extends StatelessWidget {
+  const SecondScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(AppLocalizations.of(context)!.secondScreen),
+      ),
+      body: FutureBuilder<List<dynamic>?>(
+          future: getHttp(),
+          builder: (context, snapshot) {
+            Widget child = const Center(
+              child: CircularProgressIndicator(),
+            );
+            if (snapshot.connectionState == ConnectionState.done) {
```

Comment:
If within if with reassigning child-var looks complicated - try to avoid if-nesting here.

Reply:
change this code and now it work correctly